compute/launching-an-instance/openstack-horizon.rst

```diff
@@ -1,31 +1,66 @@
-- Press "key pair" in the leftmost menu.
-- Select the SSH key you want to use (if its not already selected). This is assuming you are running a Linux instance. 
-- Press "launch instance" in the bottom right corner.
-- When the instance is provisioned it will show as status "active". 
-- By pressing its name, you are able to see its IP-configuration. There you are able to see its IP-address (both IPv4 and IPv6). To connect use either SSH or RDP (depending on image). If you are using a private subnet (and not a directly attached IP, see above), you might need to add a floating IP to be able to connect. More information in our :doc:`/networking/reaching-your-instances` article.
-
-.. Note::
-	The cloud management portal will not setup :doc:`security groups </networking/virtual-router/security-groups/index>` which will be needed to access the instance if you are using a floating IP.
+
+This will walk you through launching an :doc:`instance <../index>` from :doc:`/getting-started/managing-your-cloud/openstack-horizon`.
+
+.. note:: If you don't have a :doc:`network </networking/virtual-router/private-subnet/index>` or :doc:`../ssh-keys` available in
+          some of the steps below, you might need to do some :doc:`initial configuration </getting-started/launching-an-instance>` first.
+          An SSH-key created or added in the :doc:`/getting-started/managing-your-cloud/cloud-management-portal` is not available in Horizon as
+          they use `API users </getting-started/users.html#api-users>`__ and SSH-keys are owned by the user account and not the project.
+
+- Under **Project**, click **Compute** and then **Instances** in the left-hand side menu.
+
+- Click the **Launch an instance** button in the upper right corner.
+
+.. caution:: If you want to create an instance in the ``europe-se-1b`` availability zone you need to :doc:`create the
+             volume <../../storage/persistent-block-storage/create-volume>` in that availability zone beforehand and in
+             the **Sources** step select **Boot Source** as **Volume**. This is because we do not support attaching volumes
+             across availability zones and the instance and volume must be in the same availability zone.
+
+- Under the **Details** tab in the dialog.
+
+  - Enter a **Instance Name** your instance and optionally provide a description.
+
+  - Select an **Availability Zone**, see :doc:`Regions and Availability Zones <../regions-and-availability-zones>`.
+
+    - We recommend the default ``europe-se-1a`` availability zone if you are creating your first instance or generally
```

Comment:
As well as if you are not sure that you need to use a secondary availability zone.

(Covers cases when more AZ's may be available